src/modules/chips/components/assistChips/ElevationProfileChip.js

```diff
@@ -0,0 +1,81 @@
+import { $injector } from '../../../../injection';
+import { openProfile } from '../../../../store/elevationProfile/elevationProfile.action';
+import { AbstractAssistChip } from './AbstractAssistChip';
+import profileSvg from './assets/profile.svg';
+
+
+const Update_Profile_Coordinates = 'update_profile_coordinates';
+
+
+/**
+ *
+ * @class
+ * @author thiloSchlemmer
+ */
+export class ElevationProfileChip extends AbstractAssistChip {
```

Comment:
The component needs to overwrite `#onDisconnect` in order to ensure unsubscribing the observer (if we have one)